common-api/src/test/java/org/pmiops/workbench/db/model/CommonStorageEnumsTest.java

```diff
@@ -51,4 +55,52 @@ public void testBijectiveStorageMapping() {
       assertThat(v).isEqualTo(fromStorage.apply(storageValue));
     }
   }
+
+  // domain ID is stringly-typed so special-case this
+
+  @Test
+  public void testDomainIdBijectiveStorageMapping() {
+    for (Domain v : Domain.values()) {
+      String storageValue = CommonStorageEnums.domainToDomainId(v);
```

Comment:
CommonStorageEnums should throw IllegalArgumentException if it's not matched IMO. This assertion seems redundant; such checking is that method's job.

Reply:
I don't believe this is fixable without a wholesale rewrite of our enum system.

Which would be a good idea!  Just not in this tiny drive-by PR